ui/src/components/profile/ProfileMobileView.tsx

```diff
@@ -0,0 +1,363 @@
+                  onClick={handleClickShowOldPassword}
+                  onMouseDown={handleMouseDownPassword}
+                  edge="end"
+                >
+                  {showOldPassword ? <VisibilityOff /> : <Visibility />}
+                </IconButton>
+              </InputAdornment>
+            ),
+          }}
+        />
+        <TextField
+          fullWidth
+          id="outlined-adornment-password"
+          label="New Password"
+          style={{ marginBottom: '5%' }}
+          type={showNewPassword ? 'text' : 'password'}
+          {...register('newPassword')}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <IconButton
+                  aria-label="toggle password visibility"
+                  onClick={handleClickShowNewPassword}
+                  onMouseDown={handleMouseDownPassword}
+                  edge="end"
+                >
+                  {showNewPassword ? <VisibilityOff /> : <Visibility />}
+                </IconButton>
+              </InputAdornment>
+            ),
+          }}
+          onChange={(e) => setNewPassword(e.target.value)}
+          onBlur={onVerifyPassword}
+        />
+        {/* TODO: error handling still aint right */}
+        <TextField
+          fullWidth
+          id="outlined-basic"
+          label="Confirm New Password"
+          style={{ marginBottom: '5%' }}
+          onChange={(e) => setConfirmPassword(e.target.value)}
+          onBlur={onVerifyPassword}
+          error={!matchPassword}
+          helperText={!matchPassword ? 'Password does not match!' : ' '}
+          type={showConfirmNewPassword ? 'text' : 'password'}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <IconButton
+                  aria-label="toggle password visibility"
+                  onClick={handleClickShowConfirmNewPassword}
+                  onMouseDown={handleMouseDownPassword}
+                  edge="end"
+                >
+                  {showConfirmNewPassword ? <VisibilityOff /> : <Visibility />}
+                </IconButton>
+              </InputAdornment>
+            ),
+          }}
+        />
+        <Button variant="contained" type="submit">
+          <Typography variant="subtitle2">CHANGE PASSWORD</Typography>
+        </Button>
+      </form>
+    </Stack>
+  );
+}
+
+const ProfileMobileView: FC = (): ReactElement => {
+  const user = useSelector((state: any) => state.user);
+  const { setValue, handleSubmit, register } = useForm<UserEditorFields>();
+  const [userData, setUserData] = useState(null);
+  const [showEditProfile, setShowEditProfile] = useState<boolean>(false);
+  const [showChangePassword, setShowChangePassword] = useState<boolean>(false);
+
+  const fetchUserData = useCallback(async () => {
+    if (user.id) {
+      const { data, status } = await axios.get('/api/users/get', { params: { userId: user.id } });
```

Comment:
so.. is it userId or just id? lol

Reply:
no shd be just id. i think i forgot to change it in mobile but it shd be just id